lint/syscontainers-lint

```diff
@@ -104,6 +120,11 @@ def check_config_json(path):
         content = f.read()
 
     config = json.loads(content)
+    if config['platform']['os'] not in oses.keys():
+        log(basename, "Invalid os specified: `%s`" % config['platform']['os'], is_error=True)
+    if config['platform']['arch'] not in arches.keys():
+        log(basename, "Invalid arch specified: `%s`" % config['platform']['arch'], is_error=True)
+
```

Comment:
One minor nitpick, the user can specify a os-arch combo that is invalid, but can pass this test still since the checks are independent. E.g. "freebsd" + "mips64" (Although I doubt this would happen, just for completeness purposes?)